resources/tools.json

```diff
@@ -719,10 +719,11 @@
           "links": {"%target-dir%/simple-phpunit": "simple-phpunit"}
         },
         "sh": {
-          "command": "simple-phpunit install"
+          "command": "simple-phpunit install && SYMFONY_PHPUNIT_VERSION=8.2 simple-phpunit install"
```

Comment:
Do we need to run the install command twice?

Reply:
Yes, see https://travis-ci.com/jakzal/toolbox/jobs/208386006#L611

this ensures it installs  both v7/v8 given the current defaults of SYMFONY_PHPUNIT_VERSION (which depends on PHP version) today.

So currently you can invoke

```
simple-phpunit # 7.4
SYMFONY_PHPUNIT_VERSION=7.4 simple-phpunit
SYMFONY_PHPUNIT_VERSION=8.2 simple-phpunit
```

without any additional installation required (that's what's solved here ultimately). Invoking any other version `SYMFONY_PHPUNIT_VERSION=x.y ...` will trigger a custom installation on user's side. Nevertheless, defaults should be sane; supporting the last 2 majors.

As of november 2019, when we'll install `symfony/phpunit-bridge@4.4` we can remove it, unless we decide to keep phpunit v7 or any version wanted then.

Reply:
one thing is invoking `simple-phpunit` without setting `SYMFONY_PHPUNIT_VERSION` anywhere, let's Symfony decide the defaults (they are now bumped in https://github.com/symfony/symfony/pull/32059).

So perhaps we should always set `SYMFONY_PHPUNIT_VERSION=7|8 simple-phpunit install` explicit during install, and a default in the image as well when the user invokes `simple-phpunit`

Reply:
not sure it's big a deal for now, worst case `simple-phpunit install && SYMFONY_PHPUNIT_VERSION=8.2 simple-phpunit install` will install 8.2 twice, once symfony/symfony#32059 is merged upstream. Then we should clean it up.

Reply:
I'll have a look at this PR when I have a bit more time. Bare with me :)